src/services/keyShare.service.ts

```diff
@@ -105,7 +107,7 @@ export const getResponse = (keyShareResponseId: KeyShareValidationResponseId, er
   }
 };
 
-export const getTooltipText = (count: number) => translations.VALIDATOR.BULK_REGISTRATION.SELECTED_VALIDATORS_TOOLTIP(count);
+export const getTooltipText = (count: number, condition: boolean): string | false => condition && translations.VALIDATOR.BULK_REGISTRATION.SELECTED_VALIDATORS_TOOLTIP(count);
```

Comment:
I think you should have the move the false outside of getTooltipText to where getTooltipText is called.

Reply:
u mean move the condition outside?